client/src/components/data-entry-components/BasicAttributesFormGroup.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+
+//Component of basic attribute form elements for editor
+const BasicAttributesFormGroup = () => (
+    <React.Fragment>
+        <div className="main-alert-target"></div>
+        <h5 className="pt-1 pr-1 mr-2"><strong>Basic Information</strong></h5>
+        <div className="form-row">
+            <div className="form-group col-md-8">
+                <label for="paperTitle">Paper Title</label>
+                <input type="text" className="form-control" id="paperTitle" name="paperTitle" required="true" placeholder="required" />
+            </div>
+            <div className="form-group col-md-4">
+                <label for="doi">DOI</label>
+                <input type="text" className="form-control" id="doi" name="doi" placeholder="optional" />
+            </div>
+        </div>
+
+        <div className="form-row">
+            <div className="form-group col-md-7">
+                <label for="journalName">Journal Name</label>
+                <input type="text" className="form-control" id="journalName" name="journalName" required="true" placeholder="required" />
+            </div>
+            <div className="form-group offset-md-1 col-md-4">
+                <label for="pub_year">Year Published</label>
+                <input type="number" className="form-control" id="pubYear" name="pubYear" required="true" min="1900" Max="2100" step="1" />
+            </div>
+        </div>
+
+        <div className="form-row">
+            <div className="form-group col-md-3">
+                <label for="volume">Volume</label>
+                <input type="text" className="form-control" id="volume" name="volume" required="true" placeholder="required" />
+            </div>
+            <div className="form-group offset-md-1 col-md-3">
+                <label for="issue">Issue</label>
+                <input type="text" className="form-control" id="issue" name="issue" placeholder="optional" />
+            </div>
+            <div className="form-group offset-md-1 col-md-3">
+                <label for="series">ISSN</label>
+                <input type="text" className="form-control" id="series" name="series" placeholder="optional" />
+            </div>
+        </div>
+
+        <div className="form-row author-header">
+            <p className="pt-1 mr-2">Author(s)</p>
```

Comment:
Also this `p` tag seems to be making the author text super small. Maybe adding a font-size class to it or just plopping the text in the div tag without a p tag?

Reply:
Do we want to fix this during this PR, or have a PR for fixing the overall styling? I noticed we have some style issues in the navbar as well.

Reply:
Yeah it could be tacked in another task. We'll just need to remember to add it to our backlog.

Reply:
It should actually be fixed now. I just changed it to an `<h6>`.